### PR TITLE
 fix convention CI

### PR DESCRIPTION
**Pull request by tuanbpl-0873**
https://edu-redmine.sun-asterisk.vn/issues/11832

### CODE REVIEW

File: app/Http/Controllers/Admin/RecipeController.php [hunk 1]
```diff
@@ -86,7 +86,7 @@ public function store(Request $request)
                     $fileCount++;
     
                     if ($fileCount <= 6) {
-                        $stepImageName = $ImageStorageFolder . '/' . $stepFileName . '/' . time() . $file->getClientOriginalName();
+                        $stepImageName = $imageStorageFolder . '/' . $stepFileName . '/' . time() . $file->getClientOriginalName();
```
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/RecipeController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.Files.LineLength.TooLong</br>**Severity**: warning</br>**File**: app/Http/Controllers/Admin/RecipeController.php</br>**Line**: 89</br>Line exceeds 120 characters; contains 131 characters

File: app/Http/Controllers/Admin/RecipeController.php [hunk 2]
```diff
@@ -111,17 +111,17 @@ public function store(Request $request)
             'video_link' => $request->video,
             'level_id' => $request->level,
             'image' => $mainImageName,
-            'people_number' => $request->people_number
+            'people_number' => $request->people_number,
         ];
 
         $ingredient = [
-            'name'  => $request->ingredients
+            'name' => $request->ingredients
```
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/RecipeController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/RecipeController.php</br>**Line**: 118</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/Admin/RecipeController.php [hunk 3]
```diff
@@ -211,7 +211,7 @@ public function update(Request $request, $id)
                     $fileCount++;
     
                     if ($fileCount <= 6 && !is_null($file)) {
-                        $stepImageName = $ImageStorageFolder . '/' . $stepFileName .'/' .time() . $file->getClientOriginalName();
+                        $stepImageName = $imageStorageFolder . '/' . $stepFileName .'/' .time() . $file->getClientOriginalName();
```
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/RecipeController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.Files.LineLength.TooLong</br>**Severity**: warning</br>**File**: app/Http/Controllers/Admin/RecipeController.php</br>**Line**: 214</br>Line exceeds 120 characters; contains 129 characters

File: app/Repositories/Eloquent/RecipeRepositoryEloquent.php [hunk 4]
```diff
@@ -38,24 +38,33 @@ public function updateIngredient($id, $data = [])
...
     }
 
     public function getOneFeatureRecipe($with = [], $select = ['*'])
     {
-        $recipe = $this->model->with($with)->where('status', config('manual.recipe_status.Actived'))->orderBy('id', 'DESC')->first();
+        $recipe = $this->model
+        ->with($with)
+        ->where('status', config('manual.recipe_status.Actived'))
+        ->orderBy('id', 'DESC')
+        ->first();
```
Comment (BunnyPi04) [inline app/Repositories/Eloquent/RecipeRepositoryEloquent.php]: không tìm đc bản ghi nào đáp ứng thì trả về gì?

**Comment by tuanbpl-0873** [comment]
ready

**Pull request merged**